src/compiler/compile/render_ssr/handlers/Element.ts

```diff
@@ -113,7 +113,16 @@ export default function(node: Element, renderer: Renderer, options: RenderOption
 		}
 
 		if (name === 'group') {
-			// TODO server-render group bindings
+			const value_attribute = node.attributes.find(({ name }) => name === 'value');
+			if (value_attribute && value_attribute.chunks.length === 1) {
```

Comment:
Is there a reason you're checking for `value_attribute.chunks.length === 1`? I believe `get_attribute_value()` should work even with more complex values, and simply return an expressions that's a concatenation of strings.

Reply:
Makes sense. I initally had a bit different approach that required this check. Fixed.